apps/web/index.html

```diff
@@ -68,18 +65,64 @@
   </head>
   <body>
     <div
-      id="app"
+      id="root"
       data-h2-color="base(black)"
       data-h2-background="base(background)"
       data-h2-font-family="base(sans)"
     ></div>
-    <script src="<%= htmlWebpackPlugin.files.publicPath %>config.js"></script>
-    <noscript
-      >It looks like JavaScript is disabled in your browser settings or
-      unsupported by the browser itself. JavaScript is required to view this
-      application.<br />Il semblerait que la fonction JavaScript soit désactivée
-      ou non prise en charge dans votre navigateur comme tel. Il est nécessaire
-      d’avoir JavaScript pour consulter cette application.</noscript
-    >
+    <script>
+      const filterUnusable = (value) =>
+        !value.startsWith("$") && value.length > 0 ? value : undefined;
+
+      const data = new Map([
```

Comment:
I love that you were able to roll this in and remove that extra network roundtrip!